code/modules/surgery/bodyparts/species_parts/moth_bodyparts.dm

```diff
@@ -1,45 +0,0 @@
-/obj/item/bodypart/head/moth
```

Comment:
You shouldn't delete an entire TG file.

Reply:
Originally was the edit. But was requested to instead be moved from its original location to the new one.

Reply:
Comment it out then. We don't delete tg files.